CHANGELOG.md

```diff
@@ -19,6 +19,11 @@ Version 2.0 (under development)
   it easier for you to customize or even *localize* the theme
 * Adds author blurbs at the end of the article
 
+To Be Released
```

Comment:
Please move this patch to the top of the file.

I know 2.1 and 2.1.1 notes are not present and (under development) heading is also misleading. I need to fix that.

Reply:
Changed in next push